spec/unit/libraries/xcode_spec.rb

```diff
@@ -59,28 +59,50 @@
                    )
     end
     it 'returns the name of Xcode 10 GM when initialized with the semantic version' do
-      xcode = MacOS::Xcode.new('10.0', '/Applications/Xcode.app')
+      xcode = MacOS::Xcode.new('10.0', '/Applications/Xcode.app', '')
```

Comment:
Needing to actually pass the empty string doesn't feel like a great design, it's not clear why the Xcode class is being initialized with this value. Could we set the default value to an empty string in the initializer instead of `nil`? Alternatively, we could make the `download_url` parameter a keyword argument.

Reply:
Sure.